src/index.html

```diff
@@ -16,17 +16,42 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
+  <body class="page">
+    <!-- <input
       type="text"
       data-qa="keypress"
       placeholder="Try “Los Angeles“"
-    />
+    /> -->
 
-    <input
+    <!-- <input
       type="text"
       data-qa="hover"
       placeholder="Try “Los Angeles“"
-    />
+    /> -->
+    <div class="search-container">
```

Comment:
The `div` tag on line 31 should be properly indented with 2 spaces as per the checklist item about correct indentation between parent and child elements.